From 4eff1be9268e57d0f0c519e9eb3bce64316ff377 Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Fri, 17 Jul 2026 19:16:35 +0200
Subject: [PATCH 06308/15333] ACPI: NHLT: Remove always included kconfig.h

The inclusion of <linux/kconfig.h> is unneeded as it's guaranteed by
the build starting from the commit 2a11c8ea20bf ("kconfig: Introduce
IS_ENABLED(), IS_BUILTIN() and IS_MODULE()"). Remove it here.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Link: https://patch.msgid.link/20260717171635.1783543-1-andriy.shevchenko@linux.intel.com
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---
 include/acpi/nhlt.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/acpi/nhlt.h b/include/acpi/nhlt.h
index 2108aa6d0207..370958d93706 100644
--- a/include/acpi/nhlt.h
+++ b/include/acpi/nhlt.h
@@ -10,7 +10,6 @@
 #define __ACPI_NHLT_H__
 
 #include <linux/acpi.h>
-#include <linux/kconfig.h>
 #include <linux/overflow.h>
 #include <linux/types.h>
 
-- 
2.55.0

